dallinger/experiment_server/dashboard.py

```diff
@@ -599,3 +620,90 @@ def lifecycle():
     return render_template(
         "dashboard_cli.html", title="Experiment lifecycle Dashboard", **data
     )
+
+
+TABLE_DEFAULTS = {
+    "dom": "frtilpP",
+    "ordering": True,
+    "searching": True,
+    "select": True,
+    "paging": True,
+    "lengthChange": True,
+    "searchPanes": {"threshold": 0.99},
+}
+
+
+def prep_table_config(config):
+    """Attempts to generate a reasonable a DataTables config"""
+    table_config = deepcopy(TABLE_DEFAULTS)
+    table_config.update(deepcopy(config))
+    # Display objects and arrays in useful ways
+    for row in table_config.get("data", []):
+        for col in table_config.get("columns", []):
+            data = col["data"]
+            if isinstance(data, dict):
+                key = data.get("_")
```

Comment:
Where do these `_` characters originate? Should this be a constant?

Reply:
It's some DataTables specific nonsense, basically `_` means the default representation of the data, then there are various alternate representations like `filter` and `display`.